src/pages/en/reference/api-reference.md

```diff
@@ -379,6 +520,8 @@ export async function getStaticPaths() {
 
 The `getStaticPaths()` function should return an array of objects to determine which paths will be pre-rendered by Astro.
 
+It can also be used in [static file endpoints](/en/core-concepts/endpoints/#static-file-endpoints) and [server endpoints](/en/core-concepts/endpoints/#server-endpoints-api-routes) in `server` mode.
```

Comment:
Right, you can't use it in SSR - my [PR here](https://github.com/withastro/docs/pull/1709) tries to make that more clear!

Reply:
Yeah I don't think `getStaticPaths` make sense in SSR since we're not doing anything static. If I've implied that, my mind may have went mush when writing the docs 😬

Reply:
Ah! Right, that matches more with what I expected. So maybe we adjust this to something like this:

```suggestion
It can be used in [static file endpoints](/en/core-concepts/endpoints/#static-file-endpoints) as well as in `.astro` pages.
```